src/client/swagger/unity.yml

```
@@ -593,14 +635,47 @@ components:
     BillingNotifySettings:
       properties:
         isNotify:
+          description: opt in to threshold notifications
           type: boolean
         balanceThreshold:
           type: number
           description: threshold that when passed will trigger a notification
         notifyEmail:
           type: string
-          description: email to send billing notification
+          description: email to send notification
       required: [isNotify, balanceThreshold, notifyEmail]
+    CheckoutRequest:
+      properties:
+        paymentMethodId:
+          type: string
+          description: id from submitting the zuora form
+        street1:
+          description: for billing address
+          type: string
+        street2:
+          description: for billing address
+          type: string
+        city:
+          description: for billing address
+          type: string
+        country:
+          description: for billing address
+          type: string
+        subdivision:
+          description: for billing address (state in US)
+          type: string
+        postalCode:
+          description: for billing address
+          type: string
+        notifyEmail:
+          description: the email for the notification
+          type: string
+        balanceThreshold:
+          description: threshold for notification
+          type: string
+        notify:
+          description: whether the user will be notified about their balance
+          type: boolean
     Invite:
       properties:
         id:
```

This description should probably specify Quartz ID
```suggestion
        id:
          description: the quartz id of the invite
          readOnly: true
          type: integer
```

---

Invites only exist in Quartz and not IDPE, so it's pretty much implied that this is a Quartz ID. But not a bad idea to be explicit about it.